### PR TITLE
Clarification to Quickstart guide

### PR DESCRIPTION
**Pull request by robinpercy**
For others that need things spelled out as much as I do :)

### CODE REVIEW

File: quick-start.md [hunk 1]
```diff
@@ -58,13 +58,42 @@ Send the policy to Vault: `vault policy-write kubernetes-vault policy.hcl`
...
 
 ### 2.6 Generate the token for Kubernetes-Vault and AppID
-Generate the token: `vault token-create -role=kubernetes-vault`
+Generate the token: `vault token-create -role=kubernetes-vault`. And make a note of the token output. In the example below it would be '00000000-1111-2222-3333-444444444444'
+
+```
+$ vault token-create -role=kubernetes-vault
+
+Handling connection for 8200
+Key             Value
```
Comment (F21) [inline quick-start.md]: I think the `Handling connection for 8200` line can be removed as it's extraneous. If you run `kubectl port-forward vault-xxxxxxxx 8200` in a separate terminal, it should not be appear in the terminal you are running `vault` commands from.

File: quick-start.md [hunk 2]
```diff
@@ -58,13 +58,42 @@ Send the policy to Vault: `vault policy-write kubernetes-vault policy.hcl`
...
+---             -----
+token           00000000-1111-2222-3333-444444444444      
+token_accessor  c50fb600-aaaa-bbbb-cccc-xxxxxxxxxxxx
+token_duration  6h0m0s
+token_renewable true
+token_policies  [default kubernetes-vault]
+```
 
 Get the AppID: `vault read auth/approle/role/sample-app/role-id`
```
Comment (F21) [inline quick-start.md]: Can you add the same example for getting the AppID and adding it to the `sample-app`'s deployment as well?

**Review by F21 — CHANGES_REQUESTED** [review]
Looks great! 👍 Just a few small changes 😄

File: quick-start.md (hunk 1 above)
Comment (robinpercy) [reply quick-start.md]: Good point, removed.

File: quick-start.md (hunk 2 above)
Comment (robinpercy) [reply quick-start.md]: yep, done.

**Comment by F21** [comment]
Thanks! 👍

**Review by F21 — APPROVED** [review]

**Pull request merged**